components/Form/Form.js

```diff
@@ -0,0 +1,188 @@
+            Dosage :
+            <Input
+              type="number"
+              name="dosage"
+              id="dosage"
+              min="0"
+              step="0.01"
+            />
+          </Label>
+          <Label htmlFor="reason">
+            {" "}
+            Reason for taking:
+            <Input type="text" name="reason" />
+          </Label>
+          <Label htmlFor="form">
+            {" "}
+            Form :
+            <Select name="form">
+              <option defaultValue="Tablet">Tablet</option>
+              <option defaultValue="Capsule">Capsule</option>
+              <option defaultValue="Syrup">Syrup</option>
+              <option defaultValue="Injection">Injection</option>
+              <option defaultValue="Powder">Powder</option>
+              <option defaultValue="Ointment">Ointment</option>
+              <option defaultValue="Syringe">Syringe</option>
+              <option defaultValue="Cream">Cream</option>
+              <option defaultValue="Drops">Drops</option>
+              <option defaultValue="Spray">Spray</option>
+              <option defaultValue="Other">Other</option>
+            </Select>
+          </Label>
+          <SomeoneElse>
+            <Label htmlFor="frequency">
+              {" "}
+              Frequency :
+              <Select name="frequencyType">
+                <option defaultValue="Daily">Daily</option>
+                <option defaultValue="Weekly">Weekly</option>
+                <option defaultValue="Monthly">Monthly</option>
+              </Select>
+              <Select name="frequencyTimes">
+                <option defaultValue="Once">Once</option>
+                <option defaultValue="Twice">Twice</option>
+                <option defaultValue="Thrice">Thrice</option>
+                <option defaultValue="Four times">Four times</option>
+                <option defaultValue="Five times">Five times</option>
+                <option defaultValue="Six times">Six times</option>
+                <option defaultValue="More">More</option>
+              </Select>
+            </Label>
+            <Checkbox
+              type="checkbox"
+              name="frequencyMornings"
+              value="Mornings"
+            />
+            Mornings
+            <Checkbox
+              type="checkbox"
+              name="frequencyAfternoon"
+              value="Afternoon"
+            />
+            Afternoon
+            <Checkbox type="checkbox" name="frequencyEvening" value="Evening" />
+            Evening
+            <Checkbox type="checkbox" name="frequencyBedtime" value="Bedtime" />
+            Bedtime
+          </SomeoneElse>
+          <Label htmlFor="instractions">
+            {" "}
+            Instructions :
+            <Textarea type="text" name="instractions" />
+          </Label>
+          <Label htmlFor="refill">
+            {" "}
+            Refill :
+            <Input type="date" name="refill" />
+          </Label>
```

Comment:
```suggestion
          <Label>
            Instructions :
            <Textarea type="text" name="instructions" />
          </Label>
          <Label>
            Refill :
            <Input type="date" name="refill" />
          </Label>
```

If you nest an input in a label you do not need htmlFor. But I do not recommend to do this (see my comment above)